packages/fast-components-react-base/src/select/select.tsx

```diff
@@ -241,30 +251,74 @@ class Select extends Foundation<SelectHandledProps, SelectUnhandledProps, Select
             shouldFocusOnMount = this.props.multiselectable;
         }
         return (
-            <Listbox
-                labelledBy={this.props.labelledBy}
-                disabled={this.props.disabled}
-                focusItemOnMount={shouldFocusOnMount}
-                multiselectable={this.props.multiselectable}
-                defaultSelection={this.state.selectedItems}
-                selectedItems={this.props.selectedItems}
-                onSelectedItemsChanged={this.menuSelectionChange}
-                onItemInvoked={this.menuItemInvoked}
-                onBlur={this.handleMenuBlur}
-                managedClasses={{
-                    listbox: get(this.props.managedClasses, "select_menu", ""),
-                    listbox__disabled: get(
-                        this.props.managedClasses,
-                        "select_menuDisabled",
-                        ""
-                    ),
-                }}
+            <ViewportPositioner
+                anchor={this.rootElement}
+                {...this.getFlyoutMenuConfig()}
```

Comment:
i'd suggest re-arranging this so that anchor is below the ...this.getFlyoutMenuConfig, theres a slight chance someone could pass in "anchor" as part of the config if they arent using typescript

Reply:
yep